inst/Description.qml

```diff
@@ -3,10 +3,10 @@ import JASP.Module 	1.0
 
 Description
 {
-	title :			"Learn Bayes"
-	name: 			"jaspLearnBayes"
+	title :			qsTr("Learn Bayes")
+	name: 			qsTr("jaspLearnBayes")
```

Comment:
We don't want to translate the name of the module

Reply:
fixed